lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java

```diff
@@ -178,6 +167,13 @@ private void init(AttributeSet attrs, int defStyle) {
         mPointerHaloPaint = new Paint(Paint.ANTI_ALIAS_FLAG);
         mPointerHaloPaint.setColor(pointer_halo_color);
         mPointerHaloPaint.setStrokeWidth(mPointerRadius + 10);
+        mPointerHaloPaint.setDither(true);
+        // mPointerHaloPaint.setAlpha(150);
+
+        mPointerHaloPaint2 = new Paint(Paint.ANTI_ALIAS_FLAG);
```

Comment:
You could use just `mPointerHaloPaint2 = new Paint(mPointerHaloPaint);` which will create a new paint copying attributes already present in `mPointerHaloPaint`. Then you should just set `mPointerHaloPaint2.setStrokeWidth(mPointerRadius + 10);` to adjust proper stroke width.